.github/workflows/actions/flutter-setup/action.yml

```diff
@@ -0,0 +1,54 @@
+# IMPORTANT: FVM_VERSION, ANDROID_APP_FOLDER and IOS_APP_FOLDER have to be set on github variables
```

Comment:
add a readme file in the workflows folder, explain each file and add the requirements